native/src/App.tsx

```diff
@@ -49,7 +49,7 @@ const linking: LinkingOptions = {
 const dataContainer: DataContainer = new DefaultDataContainer()
 const store: Store<StateType, StoreActionType> = createReduxStore(dataContainer)
 
-const App = () => {
+const App = (): ReactElement => {
```

Comment:
Maybe we should add an exception for react components. The return type is always clear here.

Reply:
Not really possible for functional components: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md
Imo it would be inconsistent to a return type for functional components, but not for react components, especially we are cuttently migrating the components anyway. But feel free to proof me wrong :)